client/src/pages/Profile.js

```diff
@@ -88,146 +89,499 @@ const useStyles = makeStyles(theme => ({
         gridGap: '20px'
     },
     card: {
+        paddingBottom: '1vh',
         height: '40vh',
         width: '26vw'
     },
     cardImg: {
-        height: '31vh'
+        height: '32vh'
     },
     cardHeader: {
         marginLeft: '20px'
-    }
+    },
+    postContainer: {
+        display: 'grid',
+        gridTemplateColumns: '1fr 1fr 1fr 1fr 1fr',
+        justifyItems: 'center',
+        gridGap: '10px'
+    },
+    postImg: {
+        width: '16vw',
+        minHeight: '50vh',
+    },
+    postLink: {
+        fontSize: '16px',
+        textAlign: 'center',
+        padding: '0',
+        margin: '0',
+    },
 }));
 
 const Profile = ({ location }) => {
     const style = useStyles();
-    let [activePanel, toggle] = useState('board');
-
     const username = location.pathname.split('/')[2];
-    return (
-        <div>
-            <Navbar />
-            <div className={style.subHeader}>
-                <div className={style.nameContainer}>
-                    <img src={face} alt='' className={style.subHeaderIcon} />
+    
+    let [activePanel, toggle] = useState('board');
+    let [boards, setBoards] = useState([
+        // {
+        //     "_id": "5d4c50381de1e451e8c1760f",
+        //     "posts": [],
+        //     "date": "2019-08-08T14:20:51.333Z",
+        //     "title": "test board1",
+        //     "user": "5d4c3fcb1de1e451e8c1760d",
+        //     "__v": 0
+        // },
+        // {
+        //     "_id": "5d4c50651de1e451e8c17610",
+        //     "posts": [],
+        //     "date": "2019-08-08T14:20:51.333Z",
+        //     "title": "test board2",
+        //     "user": "5d4c3fcb1de1e451e8c1760d",
+        //     "__v": 0
+        // }
```

Comment:
same issue

Reply:
the hard coded data is the same data that would be passed into the useEffect function from the backend, its the same data i took from postman requests.

I'm not sure if the authentication is working (or I'm not sure how to activate it), but creating boards and posts isnt working for me, so I can't add these on the web app